src/Settings/Details.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Astrogoat\WhiteGlove\Settings;
+
+use Helix\Lego\Apps\Contracts\SettingsCast;
+use Helix\Lego\Http\Livewire\Traits\InteractsWithTopBar;
+use Helix\Lego\Settings\AppSettings;
+
+class Details extends SettingsCast
```

Comment:
Why the SettingsCast here and not just a Peripheral?

Reply:
@tonning I was experimenting and liked how using `SettingsCast` came with the section headers and its styling out of the box but now I see that it is the cause of the issue you mentioned on Linear about the anomaly in saving details, and so I will be getting rid of it.

Reply:
Okay, that's not the cause.

Reply:
Fixed it.